recipe/meta.yaml

```diff
@@ -80,6 +83,7 @@ requirements:
     - cuda-nvml-dev                          # [build_platform != target_platform]
     - cuda-profiler-api                      # [build_platform != target_platform]
     - libcublas-dev                          # [build_platform != target_platform]
+    - libcufile-dev                          # [build_platform != target_platform and linux]
```

Comment:
On ARM, this isn't available until CUDA 12.2. So we may want to skip for older CUDAs on ARM

This doesn't affect `x86_64`

Reply:
Thanks for letting me know. I've also noticed that I really need to pass `USE_CUFILE` only where we actually use it. I'll submit an update as soon as I finish testing.